### PR TITLE
resprites disks

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
This resprites the disks in the game

AS

## Why It's Good For The Game

Disk sprites are no longer a decade old

## Changelog
:cl:
add: new disk sprites
/:cl:

### CODE REVIEW

**Comment by rye-rice** [comment]
i do not have the will to finish this, go copy how eris does this instead

**Comment by rye-rice** [comment]
a

File: code/game/machinery/dna_scanner.dm [hunk 1]
```diff
@@ -148,16 +148,18 @@
...
 	var/list/mutations = list()
 	var/max_mutations = 6
 	var/read_only = FALSE //Well,it's still a floppy disk
 
 /obj/item/disk/data/Initialize()
 	. = ..()
+	/*shiptest edit : it conflicts with the new disks!!
 	icon_state = "datadisk[rand(0,6)]"
 	add_overlay("datadisk_gene")
+	*/
```
Comment (ZephyrTFA) [inline code/game/machinery/dna_scanner.dm]: Just remove the override entirely, you need to comment it ut

File: code/modules/disks/disk.dm [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+	label.appearance_flags = RESET_COLOR
+	. += label
+	. += protect
+	if(illustration)
+		var/mutable_appearance/writing = mutable_appearance(icon, illustration)
+		writing.appearance_flags = RESET_COLOR
+		. += writing
+
+
+/*	//unfotunately now is not the time for this
```
Comment (ZephyrTFA) [inline code/modules/disks/disk.dm]: If you don't want to add them with this PR don't have comments for them either ❤️

**Review by ZephyrTFA — APPROVED** [review]

**Pull request merged**